tests/unit/test_req_install.py

```diff
@@ -49,6 +50,14 @@ def test_forward_slash_results_in_a_link(self, tmpdir):
 
 class TestInstallRequirementFrom(object):
 
+    def test_install_req_from_string_invalid_requirement(self):
+        """
+        Requirement strings that cannot be parsed by
+        packaging.requirements.Requirement raise an InstallationError.
+        """
+        with pytest.raises(InstallationError):
+            install_req_from_req_string("http:/this/is/invalid")
```

Comment:
You might as well check the exception text since it's easy to do. Here's an example: https://github.com/pypa/pip/blob/c8e9caa8a717ea3e43dffa91b07f7cfd79e057a0/tests/unit/test_vcs.py#L280-L283

Reply:
done ✅